packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.tsx

```diff
@@ -55,13 +55,20 @@ export class MessageBar extends React.Component<IMessageBarProps, IMessageBarSta
   }
 
   private _getActionsDiv(): JSX.Element {
-    if (this.props.actions) {
-      return this.props.isMultiline ?
-        <div className='ms-MessageBar-actions'> { this.props.actions } </div> :
-        <div className='ms-MessageBar-actionsOneline'>
+    if (this.props.isMultiline) {
+      if (this.props.actions) {
+        return <div className='ms-MessageBar-actions'>
+          { this.props.actions }
+        </div>;
+      }
+    }
+    else {
+      if (this.props.actions || this.props.onDismiss) {
+        return <div className='ms-MessageBar-actionsOneline'>
```

Comment:
Agreed, but the caveat is the styling; `ms-MessageBar-actionsOneline` needs to contain both the actions and the dismiss action. My initial stab did have these methods with these responsibilities, but I thought it ended up being too circuitous for a pretty simple scenario. Happy to implement that, however, if this change is not acceptable as-is.